test-cypress/integration/setup/jdbc/jdbc.spec.js

```diff
@@ -80,7 +81,11 @@ describe('JDBC configuration', () => {
         JdbcSteps.clickOnEditButton();
         // change the query,
         YasqeSteps.clearEditor();
+
+        // The test fails intermittently, and the exact cause is unclear. One possible reason is that a popup appears over the cursor while typing, which may obstruct the input of the query.
+        // As a workaround, we will disable this feature by pasting the query directly and then typing a few spaces to mark the editor as modified.
         YasqeSteps.pasteQuery(EDIT_QUERY);
+        YasqeSteps.writeInEditor('  ');
```

Comment:
You could put this in the step. That's what they are used to

Reply:
Okay, I just wanted to run the test without affecting others.